docs/core-concepts/core-sdk/cli/overview.md

```diff
@@ -0,0 +1,121 @@
+# .NET Core Command Line Tools (CLI)
+
+## What is the .NET CLI?
+.NET Core Command Line Interface is a new foundational cross-platform toolchain for developing 
+.NET Core applications. It is "foundational" because it is the primary layer on which other, 
+higher-level tools, such as Integrated Development Environments (IDEs), editors and 
+build orchestrators can build on. 
+
+It is also cross-platform by default and has the same surface area on each of the supported platforms. This means that
+when you learn how to use the tooling and you can use it the same way from any of the supported platforms. 
+
+## Installation
+As with any tooling, the first thing is to get the tools to your machine. Depending on your scenario, you can either 
+use the native installers to install the CLI or use the installation shell script.
+
+The native installers are primarily meant for developer's machines. The CLI is distributed using each supported platform's 
+native install mechanism, for instance DEB packages on Ubuntu or MSI bundles on Windows. These installers will install 
+and set up the environment as needed for the user to use the CLI immediately after the install. However, they also 
+require administrative privileges on the machine. You can view the installation instructions on the
+[getting started page](https://aka.ms/dotnetcoregs).
+
+Install scripts, on the other hand, do not require administrative privileges. However, they will also not install any 
+prerequisites on the machine; you need to install all of the pre-requisites manually. The scripts are meant mostly for 
+setting up build servers or when you wish to install the tools without admin privileges (do note the pre-requisites 
+caveat above). You can find more information on the [install script reference topic](dotnet-install-script.md). If you are 
+interested in how to set up CLI on your CI build server you can take a look at the [CLI with CI servers](using-ci-with-cli.md) 
+document. 
+
+By default, the CLI will install in a "side-by-side" (SxS) manner. This means that multiple versions of the CLI tools 
+can coexist at any given time on a single machine. How the correct version gets used is explained in more detail in 
+the [driver section](#driver) below. 
+
+### What commands come in the box?
+The following commands are installed by default:
+
+* [new](dotnet-new.md)
+* [restore](dotnet-restore.md)
+* [run](dotnet-run.md)
+* [build](dotnet-build.md)
+* [test](dotnet-test.md)
+* [publish](dotnet-publish.md)
+* [pack](dotnet-pack.md)
+
+Of course, there is a way to import more commands on a per-project basis as well as to add your commands. This is 
```

Comment:
I'd remove the of course from here since this is not an obvious information:
Of course, there is a way -> There is also a way
to add your commands -> to add your own commands